Userland/Libraries/LibWeb/HTML/HTMLInputElement.h

```diff
@@ -193,6 +193,10 @@ class HTMLInputElement final
     JS::GCPtr<Element> placeholder_element() { return m_placeholder_element; }
     JS::GCPtr<Element const> placeholder_element() const { return m_placeholder_element; }
 
+    // https://w3c.github.io/selection-api/#dfn-has-scheduled-selectionchange-event
+    bool has_scheduled_selectionchange_event() const { return m_has_scheduled_selectionchange_event; }
+    void set_scheduled_selectionchange_event(bool value) { m_has_scheduled_selectionchange_event = value; }
```

Comment:
Would it be worth moving this to `FormAssociatedTextControlElement`? This class is used for other things in the Selection API that apply only to input and textarea elements.

Reply:
Makes sense! Done